src/node.ts

```diff
@@ -30,6 +32,7 @@ class Node extends VEventTarget {
       }
     });
     this.inputCount++;
+    this.dispatchEvent(new VEvent('addInput'));
```

Comment:
Without extra info the event handler will not have idea which input was added. Perhaps the input name should be added as extra data.